### PR TITLE
Routing, adding, updating tests

### PR DESCRIPTION
**Pull request by AymanAlqoqa**
relates#PR#79

### CODE REVIEW

File: src/controller/middleware/authentication.js [hunk 1]
```diff
@@ -2,8 +2,8 @@ exports.auth = (req, res, next) => {
   if (!req.token) {
     next();
   } else if (req.token && req.token.consultant) {
-    return res.render('consProfile');
+    return res.redirect('/consProfile');
   } else if (req.token && !req.token.consultant) {
-    return res.render('userProfile');
+    return res.redirect('userProfile');
   }
```
Comment (tomduggan85) [inline src/controller/middleware/authentication.js:6]: @AymanAlqoqa Does 'userProfile' need the forward slash too?

File: src/controller/middleware/authentication.js [hunk 2]
```diff
@@ -2,8 +2,8 @@ exports.auth = (req, res, next) => {
   if (!req.token) {
     next();
   } else if (req.token && req.token.consultant) {
-    return res.render('consProfile');
+    return res.redirect('/consProfile');
```
Comment (tomduggan85) [inline src/controller/middleware/authentication.js]: @AymanAlqoqa Just for fun, here's how you could shorten it if you wanted to:
```
if (!req.token) {
  next();
} else {
  return res.redirect( req.token.consultant ? '/consProfile' : '/userProfile' );
}
```

File: test/test.js [hunk 3]
```diff
@@ -1,7 +1,27 @@
...
+      .expect('Content-Type', 'text/html; charset=utf-8')
+      .end((errr, res) => {
+        if (err) {
+          t.error(errr);
+        } else {
+          t.equal(JSON.parse(res.text).msg, 'تم انشاء الحساب بنجاح، يمكنكي الآن تسجيل الدخول', 'must return msg of sucsses signup');
+          t.end();
+        }
+      });
+  });
```
Comment (tomduggan85) [inline test/test.js:27]: @AymanAlqoqa Looks good!

File: src/controller/middleware/authentication.js (hunk 1 above)
Comment (AymanAlqoqa) [reply src/controller/middleware/authentication.js:6]: yes

Comment (AymanAlqoqa) [reply src/controller/middleware/authentication.js:6]: ohhh

File: src/controller/middleware/authentication.js (hunk 2 above)
Comment (AymanAlqoqa) [reply src/controller/middleware/authentication.js]: okey

File: test/test.js (hunk 3 above)
Comment (AymanAlqoqa) [reply test/test.js:27]: what you suggest for this!

Comment (tomduggan85) [reply test/test.js:27]: No suggestion, I was just saying it was a good test :)

Comment (AymanAlqoqa) [reply test/test.js:27]: thanks a lot :)

**Comment by AymanAlqoqa** [comment]
@hshahwan, please review this PR as soon as possible

**Comment by AymanAlqoqa** [comment]
@all colleagues, please review this PR
thanks

**Review by naremanhilles — APPROVED** [review]

**Review by denaHS — APPROVED** [review]

**Review by alaabadra — APPROVED** [review]

**Pull request merged**